pkg/square/builder_test.go

```diff
@@ -2,253 +2,152 @@ package square_test
-
-	assert.Equal(t, set2, set1)
-}
-
-func GenerateMixedRandomTxs(t *testing.T, signer *user.Signer, rand *tmrand.Rand, normalTxCount, pfbCount int) [][]byte {
-	return shuffle(rand, GenerateOrderedRandomTxs(t, signer, rand, normalTxCount, pfbCount))
-}
-
-// TestGenerateMixedRandomTxs_Deterministic ensures that the same seed produces the same txs
-func TestGenerateMixedRandomTxs_Deterministic(t *testing.T) {
-	pfbCount := 10
-	noramlCount := 10
-	encCfg := encoding.MakeConfig(app.ModuleEncodingRegisters...)
-
-	kr := testfactory.TestKeyring(encCfg.Codec)
-	signer, err := user.NewSigner(kr, nil, testnode.TestAddress(), encCfg.TxConfig, testfactory.ChainID, 1, 0)
-	require.NoError(t, err)
-
-	rand1 := tmrand.NewRand()
-	rand1.Seed(1)
-	set1 := GenerateMixedRandomTxs(t, signer, rand1, noramlCount, pfbCount)
-
-	signer, err = user.NewSigner(kr, nil, testnode.TestAddress(), encCfg.TxConfig, testfactory.ChainID, 1, 0)
-	require.NoError(t, err)
-
-	rand2 := tmrand.NewRand()
-	rand2.Seed(1)
-	set2 := GenerateMixedRandomTxs(t, signer, rand2, noramlCount, pfbCount)
-
-	assert.Equal(t, set2, set1)
-}
-
-func shuffle(rand *tmrand.Rand, slice [][]byte) [][]byte {
-	for i := range slice {
-		j := rand.Intn(i + 1)
-		slice[i], slice[j] = slice[j], slice[i]
-	}
-	return slice
-}
+// func TestBuilderSquareSizeEstimation(t *testing.T) {
+// 	type test struct {
+// 		name               string
+// 		normalTxs          int
+// 		pfbCount, pfbSize  int
+// 		expectedSquareSize int
+// 	}
+// 	tests := []test{
+// 		{"empty block", 0, 0, 0, appconsts.MinSquareSize},
+// 		{"one normal tx", 1, 0, 0, 1},
+// 		{"one small pfb small block", 0, 1, 100, 2},
+// 		{"mixed small block", 10, 12, 500, 8},
+// 		{"small block 2", 0, 12, 1000, 8},
+// 		{"mixed medium block 2", 10, 20, 10000, 32},
+// 		{"one large pfb large block", 0, 1, 1000000, 64},
+// 		{"one hundred large pfb large block", 0, 100, 100000, appconsts.DefaultGovMaxSquareSize},
+// 		{"one hundred large pfb medium block", 100, 100, 100000, appconsts.DefaultGovMaxSquareSize},
+// 		{"mixed transactions large block", 100, 100, 100000, appconsts.DefaultGovMaxSquareSize},
+// 		{"mixed transactions large block 2", 1000, 1000, 10000, appconsts.DefaultGovMaxSquareSize},
+// 		{"mostly transactions large block", 10000, 1000, 100, appconsts.DefaultGovMaxSquareSize},
+// 		{"only small pfb large block", 0, 10000, 1, appconsts.DefaultGovMaxSquareSize},
+// 	}
+// 	for _, tt := range tests {
+// 		t.Run(tt.name, func(t *testing.T) {
+// 			rand := tmrand.NewRand()
+// 			signer, err := testnode.NewOfflineSigner()
+// 			require.NoError(t, err)
+// 			txs := generateMixedTxs(signer, rand, tt.normalTxs, tt.pfbCount, 1, tt.pfbSize)
+// 			square, _, err := square.Build(txs, appconsts.LatestVersion, appconsts.DefaultGovMaxSquareSize)
+// 			require.NoError(t, err)
+// 			require.EqualValues(t, tt.expectedSquareSize, square.Size())
+// 		})
+// 	}
+// }
```

Comment:
[blocking] is this intended to be committed commented out? Instead of merging commented code can we:
1. delete if no longer relevant
1. uncomment and fix 
2. worst case: delete with a GH issue to re-add?